src/components/modals/roles/create/index.tsx

```diff
@@ -0,0 +1,104 @@
+import {ButtonRound} from '@/components/lib'
+import {toSentence} from '@/lib/utils'
+import {createRole} from '@/pages/api/roles'
+import {FunctionComponent, useState} from 'react'
+import {Modal} from '../../modal'
+
+interface ICreateRoleModalProps {
+  isOpen: boolean
+  onClose: () => void
+  onConfirm: () => void
+}
+
+const CreateRoleModal: FunctionComponent<ICreateRoleModalProps> = ({isOpen, onClose, onConfirm}) => {
+  const [name, setName] = useState('')
+  const [canTriageRequests, setCanTriageRequests] = useState(false)
```

Comment:
Simplify - we have 8 different `useState` here; seems like one object would do the trick.